test/integration/hasMany.test.js

```diff
@@ -265,6 +265,17 @@ describe('hasMany', function () {
         limit: 1
       }]);
     });
+
+    it('should do stuff', async function() {
+      let project1 = await this.Project.findById(this.project1.id, { include: [ this.Project.associations.members ]});
+      let project2 = await this.Project.findById(this.project2.id, { include: [ this.Project.associations.members ]});
+
+      expect(project1.members, 'not to be undefined');
+      expect(project2.members, 'not to be undefined');
+      expect(project1.members.length, 'to equal', 3);
+      expect(project2.members.length, 'to equal', 4);
```

Comment:
Apply a sinon spy to `dataloader.load`, similar to [here](https://github.com/spathon/dataloader-sequelize/blob/b6f76a27576bf800dc21d55e0810ab7276ab176b/test/integration/hasMany.test.js#L56), and assert that the spy was not called